app/server/src/routes/config.ts

```diff
@@ -0,0 +1,7 @@
+import { ConfigController } from "../controllers/configController";
+
+const router = require("express").Router();
+
+router.get("/:appName", ConfigController.getConfig);
+
+export default router;
```

Comment:
Out of curiosity, is it a better approach to export  as default or module ? i.e `module.exports = router`

Reply:
eslint moans at you if you do `module.exports` from a module which does an `import`! So I guess it's considered bad practice, though I did have it that way originally and it worked fine..